_posts/2020-06-10-#19105.md

```diff
@@ -3,21 +3,68 @@ layout: pr
 date: 2020-06-10
 title: "Add Muhash3072 implementation in Python"
 pr: 19105
-authors: [fjahr]
+authors: ["sipa", "fjahr"]
 components: ["tests", "crypto"]
 host: fjahr
 status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Add notes and questions
 ## Notes
 
+- This is the second in a series of PRs implementing the Coin Statistics
+  Index. [#18000](https://github.com/bitcoin/bitcoin/pull/18000) provides
+  an overview of the work and is used to keep track of the different PRs.
+
+- The first PR in the series was [#19055](https://github.com/bitcoin/bitcoin/pull/19055)
+  and it has been featured in the PR review club [two weeks ago](https://bitcoincore.reviews/19055.html).
+  If you haven’t seen it, it would be very beneficial for you to read that
+  conversation first. There are also some links to papers in the description
+  that might help you understand the theoretical background of the algorithm
+  better.
+
+- Since then the scope of it has been reduced based on the feedback in the
```

Comment:
Based on the feedback from the last session, the scope of PR #19055 has since been reduced.